docs/getting_started.md

```diff
@@ -23,8 +23,7 @@ Use the links below to jump to a specific section:
 Before you get started, you're going to need a few things:
 
 - Your favorite IDE or text editor
-- [Python 2.7.0 or later / Python 3.6.x or
-  later](https://www.python.org/downloads/)
+- [Python Python 3.6 or later](https://www.python.org/downloads/)
```

Comment:
Python Python -> Python

Reply:
Hey, I was just being thorough!

Reply:
Haha I appreciate the strong user focus! Wouldn't want 'em to get confused ;)